### PR TITLE
Beta8

### PR DESCRIPTION
**Pull request by frinkfromthedrink**
Updates this extension to work for beta8

### CODE REVIEW

File: js/src/forum/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import app from 'flarum/app';
 import { extend } from 'flarum/extend';
 import PostUser from 'flarum/components/PostUser';
-import LevelBar from 'reflar/leven-ranks/components/LevelBar';
+import UserCard from 'flarum/components/UserCard'
```
Comment (dsevillamartin) [inline js/src/forum/index.js]: Why did you import UserCard ?

Comment (frinkfromthedrink) [reply js/src/forum/index.js]: Sorry, forgot to remove that import, was doing testing locally.

**Comment by dsevillamartin** [comment]
Fine with me if it works.

File: js/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "@reflar/enso-theme",
```
Comment (KyrneDev) [inline js/package.json]: 🤔

**Review by KyrneDev — CHANGES_REQUESTED** [review]

**Review by KyrneDev — APPROVED** [review]

**Pull request merged**